tests/test_benchmark_scenarios.py

```diff
@@ -93,6 +101,41 @@ def test_benchmark_AB_grid_pv(self, margs):
         # make sure the sum of the bus flow is always zero (there are rounding errors)
         assert df_busses_flow.net_sum.map(lambda x: 0 if x < 1e-4 else 1).sum() == 0
 
+        # make sure that installedCap is processed correctly - pv time series of results
+        # equal input pv time series times installedCap
+        # get pv input time series and evaluated period (to shorten time series)
+        input_time_series_pv = pd.read_csv(
+            os.path.join(TEST_INPUT_PATH, use_case, TIME_SERIES, "pv_solar_input.csv")
+        )
+        simulation_settings = pd.read_csv(
+            os.path.join(
+                TEST_INPUT_PATH, use_case, CSV_ELEMENTS, f"{SIMULATION_SETTINGS}.csv"
+            )
+        ).set_index("Unnamed: 0")
+        evaluated_period = float(
+            simulation_settings[SIMULATION_SETTINGS][EVALUATED_PERIOD]
+        )
+        # shorten input pv time series according to `evaluated_period`
+        input_time_series_pv_shortened = input_time_series_pv[
+            : int(evaluated_period * 24)
+        ]["kW"]
+        # get result time series and installed pv capacity
+        result_time_series_pv = df_busses_flow["pv_plant_01"]
+        energy_production_data = pd.read_csv(
+            os.path.join(
+                TEST_INPUT_PATH, use_case, CSV_ELEMENTS, f"{ENERGY_PRODUCTION}.csv"
+            )
+        ).set_index("Unnamed: 0")
+        installed_capacity = float(energy_production_data["pv_plant_01"][INSTALLED_CAP])
+        # adapt index
+        result_time_series_pv.index = input_time_series_pv_shortened.index
+
+        assert_series_equal(
```

Comment:
`assert_series_equal` seems very handy :)

Maybe import is as `pd.assert_series_equal` so that it is clear that it comes from pd?

Reply:
It's from `pandas.util.testing` so it would be a bit long `pd.util.testing.assert_series_equal`..